src/cueButton.vue

```diff
@@ -48,7 +51,7 @@ module.exports = {
 	},
 	methods: {
 		toggleCue: function() {
-			if (this.isActive()) {
+			if (this.isActive()&&this.cueMethod()!='list') {
```

Comment:
What about using a switch for the method insted? will make it easier to expand in the future and add more complicated methods if needed? Probably should break out the go and release to their own methods

Could do something like 
```
switch(this.method){
  case 'toggle':
    if(this.isActive())
      this.$root.sendCommand('cuelistRelease ' + this.$props.cue);
    else
      this.$root.sendCommand('cuelistGo ' + this.$props.cue);
    break;
  case 'list':
    this.$root.sendCommand('cuelistGo ' + this.$props.cue);
    break;
  default:
    console.log(`method ${this.method} is not valid`);
}
```

Reply:
You're absolutely right. That's definitely cleaner and allows for expansion, as you said. I've added this to the code, tested, and committed the new version.